texttospeech/cloud-client/quickstart.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Text-To-Speech API sample application .
+
+Example usage:
+    python quickstart.py
+"""
+
+
+def run_quickstart():
+    # [START tts_quickstart]
+    """Synthesizes speech from the input string of text or ssml.
+
+    Note: ssml must be well-formed according to:
+        https://www.w3.org/TR/speech-synthesis/
+    """
+    from google.cloud import texttospeech
+
+    # Instantiates a client
+    client = texttospeech.TextToSpeechClient()
+
+    # Set the text input to be synthesized
+    synthesis_input = texttospeech.types.SynthesisInput(text="Hello, World!")
+
+    # Build the voice request, select the language code ("en-US") and the ssml
+    # voice gender ("neutral")
+    voice = texttospeech.types.VoiceSelectionParams(language_code='en-US',
+                                                    ssml_gender='NEUTRAL')
```

Comment:
ssml_gender should have its own enums, please use enums to specify this field.

Reply:
done